cartography/intel/github/repos.py

```diff
@@ -214,6 +226,58 @@ def _transform_collaborators(collaborators, repo_url, transformed_collaborators)
             transformed_collaborators[user_permission].append(user)
 
 
+def _transform_python_requirements(repo_object, out_requirements_files):
+    """
+    Performs data transformations for the requirements.txt files in a GitHub repo, if available.
+    :param repo_object: The repo object.
+    :param out_requirements_files: Output array to append transformed results to.
+    :return: Nothing.
+    """
+    req_file_contents = repo_object['requirements']
+    if req_file_contents and req_file_contents.get('text'):
+        # Remove --no-binary tags if present
+        text_contents = req_file_contents['text']
+        text_contents.replace('--no-binary ', '')
```

Comment:
```suggestion
        text_contents = req_file_contents['text']
```
Non-requirements like a `--no-binary` line should already get skipped by the way we ignore lines that fail to parse as a Requirement.